### PR TITLE
Add functionality to mount host directory as data volume

### LINKED ISSUE #101: Mounting host directories as data volumes
Is there a way to specify options for  spawned containers? For example I would like to mount a host directory as a data volume (with -v option), would that be possible?

### PR DESCRIPTION
**Pull request by captainsafia**
Closes #101.

Debugging something but it's good for at least a WIP PR.

cc: @rgbkrk @koldunovn

### CODE REVIEW

**Comment by captainsafia** [comment]
Error was user related. This is ready for review.

**Comment by rgbkrk** [comment]
This does of course mean every user can change what's in those directories, deleting, adding, etc, unless one specifies them as read-only. It would be good to add that to the docs.

**Comment by captainsafia** [comment]
Hm. I'll do you one better and and a command line option for configuring permissions on each mounted directory.

**Comment by captainsafia** [comment]
Alright. Check it out, @rgbkrk.

File: dockworker.py [hunk 1]
```diff
@@ -115,9 +115,27 @@ def create_notebook_server(self, base_path, container_name, container_config):
...
+        volume_bindings = {}
+        volumes = []
+        if container_config.host_directories:
+            directories = container_config.host_directories.split(",")
+            for index, item in enumerate(directories):
+                directory = item.split(":")[0]
+                try:
+                    permissions = item.split(":")[1]
+                except IndexError:
+                    permissions = 'rw'
```
Comment (rgbkrk) [inline dockworker.py]: We probably should make our default `'ro'` if they don't specify. If there's a compelling reason why not, like Docker's default being `'rw'` (I'm assuming so), then sure.

Comment (captainsafia) [inline dockworker.py]: @rgbkrk — Yep! Docker volumes are mounted as `rw` by default. See [here](https://docs.docker.com/engine/userguide/dockervolumes/).

**Pull request merged**